test/backend_fused_op.in.cpp

```diff
@@ -757,3 +757,124 @@ NGRAPH_TEST(${BACKEND_NAME}, scale_shift)
     test_case.add_expected_output<double>(Shape{3, 6}, vector<double>(18, 6));
     test_case.run();
 }
+
+NGRAPH_TEST(${BACKEND_NAME}, fake_quantize)
```

Comment:
Can we disable the tests here instead of adding them individually to the various manifest files?

Reply:
And add a comment to enable them once FakeQuantize implementation is defined.

Reply:
The tests are in place, one of them still disabled because it's failing.